stdlib/public/core/Array.swift

```diff
@@ -1865,3 +1865,129 @@ internal struct _ArrayAnyHashableBox<Element: Hashable>
     return true
   }
 }
+
+extension Array : Differentiable where Element : AdditiveArithmetic & Differentiable {
```

Comment:
Seems this didn't fit within 80 columns. Could you make it fit?

Reply:
Will do. I was going by a 100-character column limit that I found on a Google Swift style guide online.

Reply:
Yeah, the standard library is an exception. Plus, most of the spacing rules are different from the google swift style, in the standard library.